File: app/src/main/java/it/rizzoli/atthemoment/model/principali/News.java
package it.rizzoli.atthemoment.model.principali;

import java.util.List;

public class News {
    private String Publication;
    private String Expiration;
    private String Title;
    private String Body;
    private List<String> Lines;
    private String Guid;

    public String getPublication() {
        return Publication;
    }

    public String getExpiration() {
        return Expiration;
    }

    public String getTitle() {
        return Title;
    }

    public String getBody() {
        return Body;
    }

    public List<String> getLines() {
        return Lines;
    }

    public String getGuid() {
        return Guid;
    }
}

File: model/News.swift
import Foundation
struct News: Codable {
    var publication: String
    var expiration: String
    var title: String
    var body: String
    var lines: [String]
    var guid: String

    enum CodingKeys: String, CodingKey {
        case publication = "Publication"
        case expiration = "Expiration"
        case title = "Title"
        case body = "Body"
        case lines = "Lines"
        case guid = "Guid"
    }
}
